### DISCUSSION TITLE
App-specific settings gone in version 3?

### DISCUSSION THREAD

**Discussion by come-git-it** [noah-nuebling/mac-mouse-fix#759] [Questions]
Is this functionality no longer included? I need to disable certain functionality within Cubase, which doesn't play nicely with MMF by default.

**Comment by noah-nuebling** [comment]
App-specific settings are unfortunately not available anymore in MMF 3.

 Currently, the best option is to use the Menu Bar Item to quickly disable Mac Mouse Fix for your Buttons or your scroll wheel. 
<img width="194" alt="Screenshot 2023-12-31 at 16 56 26" src="https://github.com/noah-nuebling/mac-mouse-fix/assets/40808343/df2817b3-21f1-4e05-8d27-b9af6447bd9a">


I want to bring back app-specific settings in a much nicer form, but I'm not sure when that's coming. I really do want this feature, too, though. So I plan to prioritize it.

**Reply by come-git-it** [reply]
Thanks for taking the time to respond. V3 is otherwise fantastic, and I look forward to being able to update at some point. 

I was so relieved when I found your app after jumping to Mac. Your work is very much appreciated, day after day.

**Comment by Lambozhuang** [comment]
@noah-nuebling Hi! Really love this app and can't wait to have the app-specific settings back in the future. I have a suggestion that maybe it could select apps not only by .app files, but also on-going processes (like Minecraft which is running on a java process), that would help a lot.

**Comment by cmyker** [comment]
Same here, can't use v3 due to for instance VSCode or IDEA doing their own smooth scrolling optimization thus I have to disable it for those apps, as soon as I this feature will be available I'll switch to v3 paid as well.

**Comment by 1MOHR** [comment]
Hey yeah I just wanted to add my vote in here for app specific settings. I am setting up my laptop from a factory reset, I had been using Mos before, but found your app. Yours works better except for those types of apps that just can't deal with smooth scrolling, like CAD apps and stuff. If you add whitelist/blacklist type features I'd LOVE to use this app and I'd totally pay for it.
